mysql-test/t/audit_log_many_connections.test

```diff
@@ -0,0 +1,31 @@
+#
```

Comment:
Doesn't the testcase need e.g. audit_null installed? Server has other "too many connections" testcases, but these don't currently crash at that assert?